### ISSUE TITLE
Inappropriate msconvert setting results in a lot of zero intensities

### ISSUE THREAD

**Issue by shzh0009** [Nesvilab/FragPipe#900]
- Describe the issue or question:
Dear Nesvilab,
We are analyzing LC+MS data of human cell digests by Exploris 480 using FragPipe. The software is quite fast and detect quite a lot of proteins or peptides. Thanks! We have a problem about the quantification information. We found many proteins detected with several unique peptides but they had zero intensities. I don't know the reason, and do you know how to improve it? Thanks a lot!
Best regards,
Shaochun

![unique peptides without intensity information](https://user-images.githubusercontent.com/54974363/204533141-28f0964d-3975-4439-84d7-b0f204e6d2ae.JPG)

[log_2022-11-28_10-21-55.txt](https://github.com/Nesvilab/FragPipe/files/10113100/log_2022-11-28_10-21-55.txt)

- Upload your log file
(If a log file hasn't been generated, go to the 'Run' tab in FragPipe, click 'Export Log', zip the resulting "log_[date_time].txt" file to avoid truncation, then attach the zipped file by drag & drop here.)

**Comment by fcyu** [comment]
Hi Shaochun,

It is hard to tell based on the limited information. Can you change the `min isotopes` to 1 in the `Quant (MS1)` tab since the reason might be due to the bad data quality? Also, if it is ok, can you share your data and all files in the result folder with us?

Best,

Fengchao

**Comment by shzh0009** [comment]
Hi Fengchao,
Thanks a lot for your kindly suggesting. I tried to change the min isotopes to 1 and it improved. But there are still many peptides with zero intensity. We are doing labelfree analysis of 15 samples. Follows are the results files that can be uploaded. If you need other files I can upload through google cloud because they are too big. Thanks again for your help!
Best regards,
Shaochun

[filter.log](https://github.com/Nesvilab/FragPipe/files/10120673/filter.log)
![S00045_Exploris01_SZ_Farhan_cellL6_ctrl1rep_model](https://user-images.githubusercontent.com/54974363/204735328-9e1f5a42-2f29-4168-8633-4d44d1d367d4.png)
[S00045_Exploris01_SZ_Farhan_cellL6_ctrl1rep_quant.csv](https://github.com/Nesvilab/FragPipe/files/10120674/S00045_Exploris01_SZ_Farhan_cellL6_ctrl1rep_quant.csv)



[filelist_ionquant.txt](https://github.com/Nesvilab/FragPipe/files/10120648/filelist_ionquant.txt)
[filelist_proteinprophet.txt](https://github.com/Nesvilab/FragPipe/files/10120649/filelist_proteinprophet.txt)
[log_2022-11-28_15-43-10.txt](https://github.com/Nesvilab/FragPipe/files/10120650/log_2022-11-28_15-43-10.txt)
[msbooster_params.txt](https://github.com/Nesvilab/FragPipe/files/10120651/msbooster_params.txt)

**Comment by fcyu** [comment]
Hi Shaochun,

Thank you very much for your information. Could you please upload your data (including the raw data file, fasta file, fragger.params, and log files) to https://www.dropbox.com/request/Uao29cyESlXYMq5SDS2m ? I need those files to debug and figure out the reason. We will keep the data confidential.

Best,

Fengchao

**Comment by shzh0009** [comment]
Hi Fengchao,
That is very nice that you can check those files. I already uploaded them. I converted raw data to mzML format which can be accepted by FragPipe for the analysis. If you need raw data, I will upload them later. Thanks a lot for your help and your time.
Best regards,
Shaochun

**Comment by shzh0009** [comment]
Hi Fengchao,
That is very good point. Actually I used a programed script that first converted raw data into mzML then did the database searching. I don't know the detail of parameters for the conversion. But I will confirm that with my boss tomorrow. I am going to upload several raw files. Can you have a look? Thanks a lot!
Best regards,
Shaochun

**Comment by fcyu** [comment]
Sure. Let me know after you finish the uploading.

Best,

Fengchao

**Comment by anesvi** [comment]
I think they need to confirm first and rerun if needed so we do not waste Fengchao’s time

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Comment by fcyu** [comment]
Sounds good. Then, I will wait for their confirmation.

Best,

Fengchao

**Comment by shzh0009** [comment]
Yes, thanks a lot for your time! I will confirm first! Best, Shaochun.

**Comment by shzh0009** [comment]
Hi Fengchao and all Nesvilab staff,
I confirmed with my boss and checked the detailed scripts and they do have this kind of setting for data conversion to mzML. Then I tried raw files directly but the program cracked. I don't know why but it worked for another project. Anyway, I used MSConvert manually to convert all the raw files into mzML ones using following settings. The resultant files were analysis by FragPipe for label free analysis. Now the protein of zero intensity decreased significantly. We have more than double number of proteins quantified now(before ~1500, now ~4000). So I think the problem is solved now. Thanks a lot for your excellent work and your time! Have a good day!
Best regards,
Shaochun
![mzConvert](https://user-images.githubusercontent.com/54974363/205075282-d1b377f4-795a-4966-b5e5-7a785f22bcca.JPG)
